README.md

```diff
@@ -30,8 +30,14 @@ Currently there are no binary distributions of the engine as of yet, and we are
 If you wish to attempt to build a basic visual novel with the existing C++ API, you must first install the following dependencies:
 
 ### Dependencies
-- CMake 3.19.8
-- Doxygen 1.8.17 (if building docs)
+- [CMake](https://cmake.org/download/) >= 3.19.8
+- [Vulkan SDK](https://vulkan.lunarg.com) >= 1.3.211.0 (if you require debug/validation layers)
```

Comment:
Minor tweak:

```suggestion
- [CMake](https://cmake.org/download/) >= 3.19.8
- [Vulkan SDK](https://vulkan.lunarg.com) >= 1.3.211.0
```

I also think that Line 30 of this file should probably be moved between lines 32 and 33 here.